compliance_checker/imos/imos.py

```diff
@@ -1255,35 +1255,16 @@ def check_data_variables(self, dataset):
 
     def check_quality_control_conventions_for_quality_control_variable(self, dataset):
         """
-        Check that the attributes quality_control_set and quality_control_conventions
-        are valid and consistent.
+        Check that the attribute quality_control_conventions
+        is valid and consistent.
         """
-        test_value_dict = {'1': "IMOS standard set using the IODE flags",\
-                           '2': "ARGO quality control procedure",\
-                           '3': "BOM (SST and Air-Sea flux) quality control procedure",\
-                           '4': "WOCE quality control procedure"}
+        test_value_conventions = ("IMOS standard set using the IODE flags",\
+                           "ARGO quality control procedure",\
```

Comment:
To make this easier to read, could you indent so that all items in the tuple start in the same column?